wfl/calculators/espresso.py

```diff
@@ -22,184 +25,158 @@
 from ..utils.misc import atoms_to_list
 
 # NOMAD compatible, see https://nomad-lab.eu/prod/rae/gui/uploads
-__default_keep_files = ["*.pwo"]
-__default_properties = ["energy", "forces", "stress"]
-
-
-def evaluate_autopara_wrappable(
-    atoms,
-    workdir_root=None,
-    dir_prefix="run_QE_",
-    calculator_command=None,
-    calculator_kwargs=None,
-    output_prefix="QE_",
-    properties=None,
-    keep_files="default",
-):
-    """Evaluate a configuration with Quantum Espresso
-
-    Parameters
-    ----------
-    atoms: Atoms / list(Atoms)
-        input atomic configs
+default_keep_files = ["*.pwo"]
+default_properties = ["energy", "forces", "stress"]           # done as "implemented_propertie"
+
+
+class Espresso(ase.calculators.espresso.Espresso):
+    """Extension of ASE's Espresso calculator
+
     workdir_root: path-like, default os.getcwd()
```

Comment:
Removed `workdir_root` from the docstring. `atoms` is there for every ASE calculator, so I didn't document it. Actually, I could/should just remove `atoms` from this `__init__`, because it'd be passed on as a kwarg.

Reply:
looks good